ts/components/leftPane/LeftPaneSearchHelper.tsx

```diff
@@ -298,10 +306,29 @@ export class LeftPaneSearchHelper extends LeftPaneHelper<LeftPaneSearchPropsType
     );
   }
 
-  // This is currently unimplemented. See DESKTOP-1170.
   getConversationAndMessageAtIndex(
     _conversationIndex: number
   ): undefined | { conversationId: string; messageId?: string } {
+    if (_conversationIndex < 0) {
```

Comment:
can you please rename the reference to remove the leading underscore?

Reply:
Fixed